### ISSUE TITLE
[feature request] sort and search in saved

### ISSUE THREAD

**Issue by twnaing** [aeharding/voyager#739]
The saved posts are not appeared as it was saved in chronological order.

Can we have ways to sort or search saved posts or comments?

**Comment by aeharding** [comment]
I'm changing this to just sort. Search will be more complicated and will be a generic search across all feeds, not just saved.

**Comment by sosodev** [comment]
@aeharding Will this have to use the [standard sort type](https://join-lemmy.org/api/types/SortType.html)? That's unfortunate if so because other than old/new the sort options don't seem particularly useful for saved posts. I would love a shuffle / random sort option if possible.

**Comment by aeharding** [comment]
@sosodev yes it will be the standard sort. If you'd like a shuffle/random option, I think Lemmy would be the place to request it. Once the Lemmy API supports Voyager can add it :)

**Comment by aeharding** [comment]
This is partially blocked by https://github.com/LemmyNet/lemmy/issues/3229
